sources/gbr-ons.json

```diff
@@ -19,7 +19,7 @@
 		"source_note": "ONS Open Geography portal."
 	}],
 	"url": "https://geoportal.statistics.gov.uk/",
-	"usage_concordance": -1,
+	"usage_concordance": 1,
 	"usage_geometry": -1,
```

Comment:
should this (and below) be 0?

Reply:
also no carriage return..

Reply:
I'm not sure why the `usage_geometry` was ever set to `-1`, but I'm not proposing to modify that as I don't fully understand this type of nesting. 

(I don't see the extra char return... maybe it renders differently on mobile versus desktop?)